### PR TITLE
Suggestion for tf.image.rgb to grayscale sample tutorial

### PR DESCRIPTION
**Pull request by Mbah-Javis**

### CODE REVIEW

**Comment by Mbah-Javis** [comment]
The notebook of the sample tutorial can be found [here](https://github.com/Mbah-Javis/How-to-convert-RGB-image-to-grayscale/blob/master/RBG_to_GrayScale_using_tf_image_rgb_to_grayscale.ipynb)

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: Make this a doctest (use >>>, add the outputs, and use a small example, like a 1x2 image, so people can understand what happens from reading the example)

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -1867,6 +1872,8 @@ def rgb_to_grayscale(images, name=None):
   last dimension of the output is 1, containing the Grayscale value of the
   pixels.
 
+  Sample usage with_image can be found [here on Github](https://github.com/Mbah-Javis/How-to-convert-RGB-image-to-grayscale/blob/master/RBG_to_GrayScale_using_tf_image_rgb_to_grayscale.ipynb) or [here](https://colab.research.google.com/drive/1VFbIMRKGWSknlQ_Z_d1ihqSDW08Vu117)
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: No links to random pages on github we can't ensure stay up-to-date

Comment (Mbah-Javis) [reply tensorflow/python/ops/image_ops_impl.py]: Okay. Can I submit it to tensorflow-docs tutorials that way up to date will be ensured right?
Am a participant in GCI

**Pull request closed without merging**